packages/dashboard/src/customization/pluginsConfiguration.ts

```diff
@@ -7,11 +7,21 @@ import {
   tablePlugin,
   textPlugin,
 } from './widgets';
+import { statusTimelineChartPlugin } from './widgets/statusTimeline/status-timeline.plugin';
```

Comment:
can we add this to the widgets export like the others?

Reply:
Updated